code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -386,7 +386,24 @@ meteor_act
 					adjustOxyLoss(10)
 					adjustHalLoss(5)
 
-
+				if("winded")
+					visible_message(SPAN_WARNING("[src] is winded!"), SPAN_DANGER("You feel disoriented!"))
+					confused = max(confused, 2)
+					external_recoil(40)
+					var/obj/item/item_in_active_hand = get_active_hand()
+					var/mob/living/carbon/M = user
+					if(recoil >= 60 && item_in_active_hand)
+						if(istype(item_in_active_hand, /obj/item/grab))
+							break_all_grabs(M) //See about breaking grips or pulls
+							playsound(loc, 'sound/weapons/thudswoosh.ogg', 50, 1, -1)
+							return TRUE
+						if(item_in_active_hand.wielded && !recoil >= 80)
```

Comment:
What? !recoil will be 0 if there is recoil and 1 if there is 0 recoil, it will never be higher than eighty, just use <, what the hell?

Reply:
certified copypasted Kegdo code

@Kegdo

Reply:
```suggestion
						if(item_in_active_hand.wielded && recoil < 80)
```